src/planscape/plan/models.py

```diff
@@ -71,6 +70,23 @@ class GeneratedProjectAreas(models.Model):
         null=True)
 
 
+class ScenarioSet(models.Model):
```

Comment:
Given a scenario set, how will project area polygons be retrieved?

Reply:
Via the Project FK. These map to a GeneratedProjectArea row

Reply:
Obsolete, I'll update the PR description with our newly proposed schema. We can continue discussing based on that.